src/components/Login/Web2Login.tsx

```diff
@@ -53,34 +54,37 @@ const Web2Login: FC<Props> = ({ className, walletError, onWalletSelect, setLogin
 	const [defaultWallets, setDefaultWallets] = useState<string[]>([]);
 	const [authResponse, setAuthResponse] = useState<IAuthResponse>(initAuthResponse);
 
-	const getWallet=() => {
+	const getWallet = () => {
 		const injectedWindow = window as Window & InjectedWindow;
 		setDefaultWallets(Object.keys(injectedWindow?.injectedWeb3 || {}));
 	};
 
 	const handleSubmitForm = async (data: any) => {
 		const { username, password } = data;
 
-		if(username && password) {
+		if (username && password) {
 			setLoading(true);
-			const { data , error } = await nextApiClientFetch<IAuthResponse>('api/v1/auth/actions/login', { password, username });
-			if(error || !data) {
+			const { data, error } = await nextApiClientFetch<IAuthResponse>('api/v1/auth/actions/login', { password, username });
+			if (error || !data) {
 				setError(error || 'Login failed. Please try again later.');
+				trackEvent('Login', 'Failed Login', 'Login');
 				setLoading(false);
 				return;
 			}
 
 			if (data?.token) {
 				handleTokenChange(data.token, currentUser);
-				if(isModal){
+				if (isModal) {
 					setLoading(false);
 					setLoginOpen && setLoginOpen(false);
 					return;
 				}
+				trackEvent('Login', 'Successful Login', 'Login');
```

Comment:
why 'Login' two time here in trackEvent

Reply:
one for failed, one for success. the first argument is the "Category" Because we are in the login flow, islie login. Last argument is for the button name.

Reply:
@KanishkaRajputd